cabot/cabotapp/tests/tests_basic.py

```diff
@@ -180,7 +180,7 @@ def fake_calendar(*args, **kwargs):
 
 
 def throws_timeout(*args, **kwargs):
-    raise requests.RequestException(u'фиктивная ошибка innit')
```

Comment:
mysql 3 byte limit on unicode characters (I guess I could change the charset to utf8mb4 but that doesn't seem super necessary), see http://stackoverflow.com/questions/2108824/mysql-incorrect-string-value-error-when-save-unicode-string-in-django